Cargo.toml

```diff
@@ -63,6 +63,7 @@ generic = [
   "ls",
   "mkdir",
   "mktemp",
+  "more",
```

Comment:
Windows doesn't have a ```termios.h```, which the current usage of ```more``` depends on. There may be a windows-specific way to implement ```more``` without this (and if there is, then within the more code, one could feature-gate the termios usage based on operating system), but to fix the failing travis tests for now more can just belong in the "unix" list of features.